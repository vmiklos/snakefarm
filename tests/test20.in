debug 1
out 1
time 14
load test20.map
addplayer 1
addsnake 1 1 2 1 1 0
24 0
25 0
26 0
endsnake
addplayer 2
addsnake 2 2 1 1 1 0
138 0
153 0
168 0
183 0
198 0
endsnake
addplayer 3
addsnake 3 3 2 1 1 0
58 0
59 0
endsnake
addplayer 4
addsnake 4 4 2 1 1 0
140 0
141 0
endsnake
addplayer 5
addsnake 5 5 2 1 1 0
96 0
97 0
98 0
endsnake
addplayer 6
addsnake 6 6 1 1 1 0
174 0
189 0
204 0
endsnake
addplayer 7
addsnake 7 7 0 1 1 0
148 0
147 0
146 0
145 0
130 0
endsnake
addplayer 8
addsnake 8 8 3 1 1 0
118 0
103 0
88 0
73 0
72 0
endsnake
step
turnright 7
turnright 8
step
turnright 7
turnright 8
step
turnleft 6
turnright 7
turnright 8
step
turnleft 1
turnright 6
step
turnright 6
step
turnright 2
step
step
turnleft 1
turnleft 6
step
step
turnleft 6
step
step
