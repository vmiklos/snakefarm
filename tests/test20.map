1;W;0;0;
2;W;0;1;
3;W;0;2;
4;W;0;3;
5;W;0;4;
6;W;0;5;
7;W;0;6;
8;W;0;7;
9;W;0;8;
10;W;0;9;
11;W;0;10;
12;W;0;11;
13;W;0;12;
14;W;0;13;
15;W;0;14;

16;W;1;0;
17;0;2;16;
18;0;3;17;
19;0;4;18;
20;A;5;19;
21;0;6;20;
22;T;7;21;
23;T;8;22;
24;0;9;23;
25;0;10;24;
26;0;11;25;
27;0;12;26;
28;0;13;27;
29;0;14;28;
30;W;15;29;

31;W;16;0;
32;0;17;31;
33;0;18;32;
34;0;19;33;
35;0;20;34;
36;0;21;35;
37;0;22;36;
38;0;23;37;
39;0;24;38;
40;0;25;39;
41;0;26;40;
42;0;27;41;
43;0;28;42;
44;0;29;43;
45;W;30;44;

46;W;31;0;
47;0;32;46;
48;0;33;47;
49;0;34;48;
50;0;35;49;
51;0;36;50;
52;0;37;51;
53;0;38;52;
54;0;39;53;
55;0;40;54;
56;A;41;55;
57;0;42;56;
58;0;43;57;
59;0;44;58;
60;W;45;59;

61;W;46;0;
62;0;47;61;
63;0;48;62;
64;0;49;63;
65;F;50;64;
66;0;51;65;
67;0;52;66;
68;0;53;67;
69;0;54;68;
70;0;55;69;
71;0;56;70;
72;0;57;71;
73;0;58;72;
74;0;59;73;
75;W;60;74;

76;W;61;0;
77;0;62;76;
78;0;63;77;
79;0;64;78;
80;0;65;79;
81;0;66;80;
82;0;67;81;
83;0;68;82;
84;0;69;83;
85;0;70;84;
86;0;71;85;
87;0;72;86;
88;0;73;87;
89;0;74;88;
90;W;75;89;

91;W;76;0;
92;0;77;91;
93;0;78;92;
94;F;79;93;
95;0;80;94;
96;0;81;95;
97;0;82;96;
98;0;83;97;
99;0;84;98;
100;0;85;99;
101;0;86;100;
102;0;87;101;
103;0;88;102;
104;0;89;103;
105;W;90;104;

106;W;91;0;
107;0;92;106;
108;0;93;107;
109;0;94;108;
110;0;95;109;
111;0;96;110;
112;0;97;111;
113;0;98;112;
114;0;99;113;
115;0;100;114;
116;0;101;115;
117;F;102;116;
118;0;103;117;
119;0;104;118;
120;W;105;119;

121;W;106;0;
122;0;107;121;
123;0;108;122;
124;0;109;123;
125;0;110;124;
126;0;111;125;
127;0;112;126;
128;0;113;127;
129;0;114;128;
130;0;115;129;
131;0;116;130;
132;A;117;131;
133;0;118;132;
134;0;119;133;
135;W;120;134;

136;W;121;0;
137;0;122;136;
138;0;123;137;
139;A;124;138;
140;0;125;139;
141;0;126;140;
142;0;127;141;
143;0;128;142;
144;0;129;143;
145;0;130;144;
146;0;131;145;
147;0;132;146;
148;0;133;147;
149;0;134;148;
150;W;135;149;

151;W;136;0;
152;0;137;151;
153;0;138;152;
154;0;139;153;
155;0;140;154;
156;0;141;155;
157;0;142;156;
158;0;143;157;
159;F;144;158;
160;0;145;159;
161;0;146;160;
162;0;147;161;
163;0;148;162;
164;0;149;163;
165;W;150;164;

166;W;151;0;
167;0;152;166;
168;0;153;167;
169;0;154;168;
170;0;155;169;
171;0;156;170;
172;0;157;171;
173;0;158;172;
174;0;159;173;
175;0;160;174;
176;0;161;175;
177;0;162;176;
178;0;163;177;
179;0;164;178;
180;W;165;179;

181;W;166;0;
182;0;167;181;
183;0;168;182;
184;0;169;183;
185;0;170;184;
186;0;171;185;
187;0;172;186;
188;0;173;187;
189;0;174;188;
190;0;175;189;
191;0;176;190;
192;0;177;191;
193;0;178;192;
194;0;179;193;
195;W;180;194;

196;W;181;0;
197;0;182;196;
198;0;183;197;
199;0;184;198;
200;0;185;199;
201;0;186;200;
202;0;187;201;
203;0;188;202;
204;0;189;203;
205;0;190;204;
206;0;191;205;
207;0;192;206;
208;0;193;207;
209;0;194;208;
210;W;195;209;

211;W;196;0;
212;W;197;211;
213;W;198;212;
214;W;199;213;
215;W;200;214;
216;W;201;215;
217;W;202;216;
218;W;203;217;
219;W;204;218;
220;W;205;219;
221;W;206;220;
222;W;207;221;
223;W;208;222;
224;W;209;223;
225;W;210;224;
