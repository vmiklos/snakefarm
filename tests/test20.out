Event MapLoad test20.map
Event Step 1
StepEvent Snake_StoneBerry
End Step
Event Step 2
End Step
Event Step 3
End Step
Event Step 4
StepEvent Snake_SawBerry
End Step
Event Step 5
End Step
Event Step 6
StepEvent Snake_FieldBerry
End Step
Event Step 7
End Step
Event Step 8
End Step
Event Step 1
StepEvent Snake_StoneBerry
End Step
Event StoneStep 1
Event Step 2
End Step
Event Step 3
StepEvent Snake_SawBerry
End Step
Event Step 4
StepEvent Snake_Snake 2
StepEvent SnakeSplit 3 9
End Step
Event Step 5
StepEvent Snake_FieldBerry
End Step
Event Step 6
End Step
Event Step 7
StepEvent TurnRight
End Step
Event Step 8
StepEvent TurnRight
StepEvent Snake_SawBerry
End Step
Event Step 1
End Step
Event StoneStep 1
Event StoneStep 1
Event Step 2
End Step
Event Step 9
End Step
Event Step 3
End Step
Event Step 4
End Step
Event Step 5
StepEvent Snake_Snake 2
StepEvent SnakeDie 5
End Step
Event Step 6
End Step
Event Step 7
StepEvent TurnRight
End Step
Event Step 8
StepEvent TurnRight
StepEvent Snake_FieldBerry
End Step
Event Step 1
StepEvent Snake_SawBerry
End Step
Event Step 2
End Step
Event Step 9
StepEvent Snake_Wall
StepEvent SnakeDie 9
End Step
Event Step 3
End Step
Event Step 4
StepEvent Snake_Wall
StepEvent SnakeDie 4
End Step
Event Step 6
StepEvent TurnLeft
End Step
Event Step 7
StepEvent TurnRight
StepEvent Snake_Snake 7
StepEvent SnakeDie 7
End Step
Event Step 8
StepEvent TurnRight
StepEvent Snake_Snake 8
StepEvent SnakeSplit 4 10
End Step
Event Step 1
StepEvent TurnLeft
End Step
Event Step 2
End Step
Event Step 3
End Step
Event Step 6
StepEvent TurnRight
End Step
Event Step 8
End Step
Event Step 10
End Step
Event Step 1
End Step
Event Step 2
End Step
Event Step 3
End Step
Event Step 6
StepEvent TurnRight
End Step
Event Step 8
StepEvent Snake_Wall
StepEvent SnakeDie 8
End Step
Event Step 10
StepEvent Snake_Wall
StepEvent SnakeDie 10
End Step
Event Step 1
StepEvent Snake_FieldBerry
End Step
Event StoneStep 1
Event StoneStep 1
Event Step 2
StepEvent TurnRight
End Step
Event Step 3
End Step
Event Step 6
End Step
Event Step 1
End Step
Event Step 2
StepEvent Snake_Snake 1
StepEvent SnakeDie 2
End Step
Event Step 3
StepEvent Snake_Snake 1
StepEvent SnakeDie 3
End Step
Event Step 6
End Step
Event Step 1
StepEvent TurnLeft
End Step
Event Step 6
StepEvent TurnLeft
End Step
Event Step 1
End Step
Event Step 6
End Step
Event Step 1
End Step
Event Step 6
StepEvent TurnLeft
End Step
Event Step 1
End Step
Event Step 6
StepEvent Snake_Snake 1
StepEvent SnakeDie 6
End Step
